adr/2022-08-decouple-from-ietf.md

```diff
@@ -14,16 +14,16 @@ Related Issues:
 
 ## Context and Problem Statement
 
-Currently JSON Schema follows the IETF Internet-Draft (I-D) process for spec
-development and releases but aren't associated with IETF in any way. Our
+Currently JSON Schema loosely follows the IETF Internet-Draft (I-D) process for
+spec development and releases but isn't associated with IETF in any way. Our
 perceived involvement with IETF causes confusion and misunderstanding within our
-community in the cases were our needs and the realities of our situation deviate
-from the IETF process.
+community in the cases were our practices and the realities of our situation
```

Comment:
I'm pretty sure the only way our specification-writing practices deviate from anything is that we take too long between drafts, which expire after 6 months.  But that's actually pretty common, even if we have a relatively extreme case.

The "realities of our situation" part (with broad adoption for over a decade despite being an individual draft, and with the lack of suitability of what we've been told would happen if we joined a working group) are the real issues here.